src/test/java/io/kestra/plugin/weaviate/SchemeCreateTest.java

```diff
@@ -0,0 +1,62 @@
+package io.kestra.plugin.weaviate;
+
+import io.kestra.core.runners.RunContext;
+import io.kestra.core.runners.RunContextFactory;
+import io.micronaut.test.extensions.junit5.annotation.MicronautTest;
+import jakarta.inject.Inject;
+import org.junit.jupiter.api.Test;
+
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+
+@MicronautTest
+public class SchemeCreateTest {
+
+    public static final String SCHEME = "http";
+    public static final String HOST = "localhost:8080";
+
+    @Inject
+    private RunContextFactory runContextFactory;
+
+    @Test
+    public void testSchemeCreate() throws Exception {
+        RunContext runContext = runContextFactory.of();
+
+        String className = "TestSchema_"+ UUID.randomUUID().toString().replace("-", "").toLowerCase();
+        Map<String, List<String>> parameters = Map.of("title", List.of("text"));
+
+        SchemaCreate.Output schemeOutput = SchemaCreate.builder()
+            .scheme(SCHEME)
+            .host(HOST)
+            .className(className)
+            .parameters(parameters)
+            .build()
+            .run(runContext);
+
+        assertThat(schemeOutput.getSuccess(), is(Boolean.TRUE));
+
+        Query.Output queryOutput = Query.builder()
+            .scheme(SCHEME)
+            .host(HOST)
+            .query("""
+                   {
+                          Get {
+                            %s (
+                              limit: 10
+                            ) {
+                              title
+                            }
+                          }
+                        }
+                   """.formatted(className))
+            .build()
+            .run(runContext);
+
+        Map<String, Object> data = queryOutput.getData();
+        assertThat(data.size(), is(1));
```

Comment:
How is there 1 row while you only create the schema there ? isn't it supposed to create the metadata but no entity there ?

Reply:
When create schema it creates just class and fields that class includes in him self. But if we will try to retrieve this, it will give empty object only classname will be in response, but if schema was not created, then error will be thrown (on get request)

Reply:
It is returned Map where key is class name but value is empty list. (Actually it is responds with map where key is string "Get", value is map where key is class name, and etc.)

Reply:
Can you please assert the <className, emptyList> entry then ?

Reply:
to assert that we don't create an empty object :+1: